edge-modules/edge-dashboard/src/health.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#[derive(Debug)]
+pub enum Health {
+    Healthy,
+    Degraded,
+    Poor,
+}
+
+#[derive(Debug)]
+pub struct Status {
+    iotedged: bool,
+    edge_agent: bool,
+    edge_hub: bool,
+    other_modules: bool,
+}
+
+impl Status {
+    pub fn new() -> Self {
```

Comment:
Do you ever use the default values here? If not, you can have the parameters be in the constructor.